src/lib/cht-api.ts

```diff
@@ -182,20 +189,28 @@ export class ChtApi {
       include_docs: true,
     };
     console.log('axios.get', url, params);
-    const resp = await this.axiosInstance.get(url, { params });
+    let places: any[] | undefined = ChtApi.cache.get(placeType);
```

Comment:
previously cache was keyed by `[domain][contact-type]` but now only by `[contact-type]`.  Isn't the domain dimension required? This allowed independant caches and timers for each county eCHIS instance... just because the cache has expired for Nairobi county instance, should it also expire for Migori county instance?